app/styles/app.scss

```diff
@@ -1,5 +1,7 @@
 @import 'variables';
 @import 'ember-paper';
+@import "ember-paper-expansion-panel";
```

Comment:
👍 I was gonna say, I got `Error: File to import not found or unreadable: ember-paper-expansion-panel.`

Also... if the table styles should be there, I wonder why they're not?